README.md

```diff
@@ -1,7 +1,7 @@
 <h2 align="center">
   <a href="http://mlpack.org"><img
 src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="background-color:rgba(0,0,0,0);" height=230 alt="mlpack: a fast, flexible machine learning library"></a>
-  <br>a fast, flexible machine learning library<br>
+  <br>a fast, header-only machine learning library<br>
```

Comment:
What is the reason to remove the flexible part? I still think mlpack is flexible.

Reply:
too many qualifiers

Reply:
I missed the other comment.